app/models/supplemental_file.rb

```diff
@@ -20,6 +20,8 @@ class SupplementalFile < ApplicationRecord
 
   serialize :tags, Array
 
+  attr_accessor :language
```

Comment:
This accessor is here so that the program does not complain about SupplementalFiles not having a language method and so that a language can be assigned in the test suite. This will be deprecated by the multiple captions PR which adds a database migration that adds a language field to SupplementalFile.

So, if this gets merged first we will need to rebase the Multiple Captions PR and vice versa.